netlify.toml

```
@@ -278,6 +278,10 @@
   from = "/docs/third-party-tools/kafka/"
   to = "/docs/third-party-tools/kafka/overview/"
 
+[[redirects]]
+  from = "/blog/2020-11-26-why-timeseries-data/"
```

```suggestion
  from = "/blog/2020/11/26/why-timeseries-data/"
```

The date part is separated by forward slash. You can confirm this by checking other rules in this file that redirect blog post pages or other blog posts, like, for example [this one](https://deploy-preview-1307--unruffled-blackwell-31bfb2.netlify.app/blog/2023/01/03/change-data-capture-with-questdb-and-debezium) (notice how URL is separated with forward slash for date part)